src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/EncoderParameter.cs

```diff
@@ -91,7 +91,7 @@ public EncoderParameter(Encoder encoder, byte value)
 
             _parameterValueType = EncoderParameterValueType.ValueTypeByte;
             _numberOfValues = 1;
-            _parameterValue = Marshal.AllocHGlobal(Marshal.SizeOf(typeof(byte)));
+            _parameterValue = Marshal.AllocHGlobal(sizeof(byte));
```

Comment:
Nit: You can also replace all `Marshal.WriteByte`s with direct pointer writes, e.g. `*(byte *)_parameterValue = value;`.